protocols/generic/generic.go

```diff
@@ -36,7 +36,16 @@ import (
 )
 
 const (
-	blockchain = "generic"
+	blockchain		= "generic"
+	p2pPort			= 9000
+)
+
+type topology string
+
+const (
+	all       = topology("all")
+	sequence  = topology("sequence")
+	randomTwo = topology("randomTwo")
```

Comment:
i thought we were going to have 4 options
1. all
2. serialized
3. paired
4. random